setup.py

```diff
@@ -32,7 +32,10 @@
         "datadog==0.39.0",
         "ddtrace==0.45.0",
         "wrapt==1.11.2",
-        "setuptools==42.0.2",
+        # If building for Python 3, use the latest version of setuptools
+        "setuptools==54.2.0; python_version >= '3.0'",
```

Comment:
Should we loosen the version requirement so it's a greater than? I think part of the issue was we lock to a particular version, which the user might not be on.

Reply:
That sounds good to me, I'm not sure how we'd choose which version though.

I think this would fix this particular customer's issue as is, since the conflict they were getting was
```
The conflict is caused by:
  datadog-lambda 2.26.0 depends on setuptools==42.0.2
  wiremock 2.1.3 depends on setuptools>=45.2.0
```

Which should be fixed by us pinning anything >=45.2.0.